source/js/components/search-form.js

```diff
@@ -1,98 +1,106 @@
 import fetchUtils from "@degjs/fetch-utils";
+import formMapper from "@degjs/form-mapper";
 
 const searchForm = () => {
 
 
     const formEl = document.querySelector('.js-search-form');
-    const layoutEl = document.querySelector('.js-general-content')
+    const layoutEl = document.querySelector('.js-general-content');
+    const searchInputEl = formEl.querySelector('.js-search');
+    const searchButtonEl = formEl.querySelector('.js-search-btn');
 
     const url = formEl.dataset.api;
 
     function init() {
-        formEl.addEventListener('submit', e => {
-            e.preventDefault();
-            getData();
-        });
+        formEl.addEventListener('submit', searchSubmit);
     }
 
+    function searchSubmit(e) {
+        e.preventDefault();
+        getData();
+    }
 
     function getData() {
-        const formEls = formEl.querySelectorAll('.js-value');
+        preloader(true);
         fetchUtils.getJSON(
             url,
             {
-                body: JSON.stringify(formEls),
+                body: JSON.stringify(formMapper.getValues(searchInputEl)),
```

Comment:
[suggestion] See previous comment about placement of this causing a re-querying of the DOM, but also, consider not loading in an entire library when you're really just getting the value of a single element. formMapper is great for more complex forms, but you could do this without formMapper at all, just by using `searchInputEl.value`.

Reply:
yep Fixed. Thanks.